eng/pipelines/templates/Functional_Tests_On_Windows.yml

```diff
@@ -1,4 +1,12 @@
 steps:
+- task: PowerShell@1
+  displayName: "Set MSBuildEnableWorkloadResolver to false"
+  # Set MSBuildEnableWorkloadResolver to work around https://github.com/dotnet/sdk/issues/17461
```

Comment:
Is this something we'd want to unset? 
Would having this setting cause any problems running these tests locally?

Reply:
This env var is not set locally and the test passes.
If I set `MSBuildEnableWorkloadResolver` to true locally, the test (e.g. `AddPkg_ConditionalAddWithUserInputFramework_Success`) fails for the same reason:
```
Message: 
System.AggregateException : One or more errors occurred.
---- Microsoft.Build.Exceptions.InvalidProjectFileException : The SDK 'Microsoft.NET.SDK.WorkloadAutoImportPropsLocator' specified could not be found.  C:\Program Files\dotnet\sdk\5.0.409\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.props

  Stack Trace: 
Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
Task`1.GetResultCore(Boolean waitCompletionNotification)
Task`1.get_Result()
<AddPkg_ConditionalAddWithUserInputFramework_Success>d__17.MoveNext() line 673
--- End of stack trace from previous location where exception was thrown ---
TaskAwaiter.ThrowForNonSuccess(Task task)
TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
--- End of stack trace from previous location where exception was thrown ---
TaskAwaiter.ThrowForNonSuccess(Task task)
TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
<17 more frames...>
Evaluator`4.Evaluate(IEvaluatorData`4 data, ProjectRootElement root, ProjectLoadSettings loadSettings, Int32 maxNodeCount, PropertyDictionary`1 environmentProperties, ILoggingService loggingService, IItemFactory`2 itemFactory, IToolsetProvider toolsetProvider, ProjectRootElementCacheBase projectRootElementCache, BuildEventContext buildEventContext, ISdkResolverService sdkResolverService, Int32 submissionId, EvaluationContext evaluationContext, Boolean interactive)
ProjectImpl.Reevaluate(ILoggingService loggingServiceForEvaluation, ProjectLoadSettings loadSettings, EvaluationContext evaluationContext)
ProjectImpl.ReevaluateIfNecessary(ILoggingService loggingServiceForEvaluation, ProjectLoadSettings loadSettings, EvaluationContext evaluationContext)
ProjectImpl.ReevaluateIfNecessary(EvaluationContext evaluationContext)
ProjectImpl.Initialize(IDictionary`2 globalProperties, String toolsVersion, String subToolsetVersion, ProjectLoadSettings loadSettings, EvaluationContext evaluationContext)
Project.ctor(ProjectRootElement xml, IDictionary`2 globalProperties, String toolsVersion, String subToolsetVersion, ProjectCollection projectCollection, ProjectLoadSettings loadSettings, EvaluationContext evaluationContext)
Project.ctor(ProjectRootElement xml, IDictionary`2 globalProperties, String toolsVersion)
MSBuildAPIUtility.GetProject(String projectCSProjPath, IDictionary`2 globalProperties) line 71
MSBuildAPIUtility.AddPackageReferencePerTFM(String projectPath, LibraryDependency libraryDependency, IEnumerable`1 frameworks) line 148
<ExecuteCommand>d__0.MoveNext() line 240
```

Reply:
workloads are net to .NET 6, so they didn't exist in the .NET 5 SDK.  At this time, I believe it's only used by maui and xamarin, which we don't have tests for at this time, hence it doesn't break us (yet?).

Reply:
I tried to remove this workaround. There are multiple tests in NuGet.XPlat.FuncTest.XPlatAddPkgTests failed.
The build is: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6304121&view=results